packages/@aws-cdk/aws-servicecatalogappregistry/README.md

```diff
@@ -73,12 +73,10 @@ and want to associate all stacks in the `App` scope to `MyAssociatedApplication`
 ```ts
 const app = new App();
 const associatedApp = new appreg.ApplicationAssociator(app, 'AssociatedApplication', {
-    applicationName: 'MyAssociatedApplication',
-    description: 'Testing associated application',
-    stackProps: {
-        stackName: 'MyAssociatedApplicationStack',
-        env: {account: '123456789012', region: 'us-east-1'},
-    },
+  applications: [appreg.ApplicationBuilder.createApplication('MyAssociatedApplication', {
```

Comment:
I'm not sure `ApplicationBuilder` covers the intent. Is this an instance of the *Builder Pattern*? No, so calling it a `Builder` will raise incorrect expectations. All the more confusing given that in Java, we do generate builder patterns for CDK classes. So now you have builders, and builders, and they're not the same.


Some alternative names:

* ApplicationSource
* ApplicationStack
* TargetApplication
* AssociatableApplication
* AssociationTarget
* Association
* ...